assets/js/components/Dashboard/AlertsPage.tsx

```diff
@@ -37,28 +38,21 @@ interface AlertsResponse {
   screens_by_alert: ScreensByAlert;
 }
 
-interface Props {
-  places: Place[];
-  isVisible: boolean;
-}
-
-const AlertsPage: ComponentType<Props> = (props: Props) => {
-  const { places, isVisible } = props;
+const AlertsPage: ComponentType = () => {
+  const { places } = useOutletContext<{ places: Place[] }>();
 
   const [alerts, setAlerts] = useState<Alert[]>([]);
   const [screensByAlertMap, setScreensByAlertMap] = useState<ScreensByAlert>(
```

Comment:
We eventually plan to move this state up as well, right? (Just double checking)

Reply:
Yep! I was just going to leave that out of scope for this particular task. It will be done in URL change part.